bridges/BooruprojectBridge.php

```diff
@@ -14,13 +14,16 @@ class BooruprojectBridge extends GelbooruBridge {
 				'type' => 'number'
 			),
 			't' => array(
-				'name' => 'tags'
+				'name' => 'tags',
+				'required' => true,
+				'exampleValue'	=> 'tagme'
```

Comment:
```suggestion
				'exampleValue'	=> 'tagme',
				'title'	=> 'Use "all" to get all posts'
```
While the parent classes return all posts with tag=<blank>, booru.org seems to require tag=all to get the same content (the front page of the booru).

Reply:
Sure.